Docs/Guide.md

```diff
@@ -44,9 +44,9 @@ Then, create a partition on the remaining space. By the end, your partition map
 
 ![Windows XP partition setup](Assets/Guide/QEMU_002.png)
 
-Now, you can install to the larger partition and go through the installation as normal. This should only take a few minutes unless you don't type your product key in correctly 37 times.
+Now, you can install to the larger partition and go through the installation as normal. This should only take a few minutes unless you type in your product key incorrectly 37 times.
 
-Once XP is installed, navigate to File Explorer and format the extra partition we created as FAT32 A quick format is complete. This will act as the EFI system partition and is where the bootloader will be stored. Once you do that, shut down the system.
+Once XP is installed, navigate to File Explorer and format the extra partition we created as FAT32 (use Quick Format). This will act as the EFI system partition and is where the bootloader will be stored. Once you do that, shut down the system.
```

Comment:
Alternatively, if you would like it more:
```suggestion
Once XP is installed, navigate to File Explorer and perform a quick format of the extra partition we created as FAT32. This will act as the EFI system partition and is where the bootloader will be stored. Once you do that, shut down the system.
```